dbms/tests/queries/0_stateless/00670_truncate_temporary_table.sql

```diff
@@ -0,0 +1,16 @@
+drop table if exists test;
+create temporary table test(id int);
+
+select '======Before Truncate======';
+insert into test values(0);
+select * from test;
+
+select '======After Truncate And Empty======';
+truncate table test;
```

Comment:
truncate temporary table  test

Reply:
`temporary` isn't needed if no database prefix is specified.

Reply:
You can refer to https://github.com/zhang2014/ClickHouse/blob/960fad130ca68dc6e9e5fb282a541d3a780c2200/dbms/tests/queries/0_stateless/00564_show_tables_with_temporary.sql, They should be consistent : )